pkg/ipam/init.go

```diff
@@ -41,17 +42,17 @@ func nextIP(ip net.IP) {
 }
 
 func reserveLocalRoutes(ipam *Config) {
-	log.Debugf("Checking local routes for conflicts...")
+	log.Debug("Checking local routes for conflicts...")
 
 	link, err := netlink.LinkByName("cilium_host")
 	if err != nil || link == nil {
-		log.Warningf("Unable to find net_device cilium_host: %s", err)
+		log.WithError(err).Warn("Unable to find net_device cilium_host")
```

Comment:
This is a serious error and indicate a bug, let's change it to `Fatal()`

Reply:
Done. Note that I removed the return after it since Fatal exits (I'm not sure if leaving the return triggers any linters)